build_runner/README.md

```diff
@@ -83,22 +151,29 @@ from the current package.
 In general it is best to be as specific as possible with your `InputSet`s,
 because all matching files will be provided to `declareOutputs`.
 
-### Outputs
+> Future versions of `build_runner` may aid in automatically generating
+> `InputSet`s based on the builders being used! See [issue #353][issue_353].
 
-You may only output files in the current package, but anywhere in the current
-package is allowed.
-
-You are not allowed to overwrite existing files, only create new ones.
+### Outputs
 
-Outputs from previous builds will not be treated as inputs to later ones.
+* You may output files anywhere in the current package.
```

Comment:
Lets add a line about other packages as well, "If setting `writeToCache: true`, you can output files under the `lib` folder of any package you depend on."

Reply:
Done!